### DISCUSSION TITLE
Trained ckpt file can only generate character image, while neglecting other components

### DISCUSSION THREAD

**Discussion by henryvii99** [d8ahazard/sd_dreambooth_extension#560] [Q&A] [answered]
Hello all, I am now playing with dreambooth with the aim of recreating anime characters.
It does successfully with what I wanted, but go horribly wrong with background or maybe some complex character poses.

For example, I have a portrait image of Miho yesterday:
https://imgbox.com/hddCdM8P

By using only this image for Dreambooth training, 
and about 3000 images in regularization, in which they are all
portrait images with same dimension as Miho, but with different degree of variations.

It generates something like this:
https://imgbox.com/pq3mwKUW

Facial details are perfectly replicated, but background are bland and sometimes missing.
Background problem becomes severe (at 900 steps and above) and clothing becomes mosaic like patterns.

Is there something I can do? Should I mess with regularization images, or something related to parameters?

**Comment by cerega66** [comment, marked as answer]
The best option for you is to give some kind of background to your image for training. You can also try to add a "detailed background" to the prompt on the finished models, and a "simple background" in negative.

**Reply by henryvii99** [reply]
That is, to put desired background images in regularization folder as well?

**Reply by cerega66** [reply]
No. Your character image must include a detailed background. The image you provided has a character with a simple background, which causes the "simple background" token to be added to your prompt when you try to generate the image. However, when generating, you can specify the opposite token, "detailed background".

**Reply by henryvii99** [reply]
Many thanks for your help!
